0000-interledger-protocol-4/0000-interledger-protocol-4.md

```diff
@@ -0,0 +1,159 @@
+- **Condition and Fulfillment** - ILP packets are secured with conditions so that they cannot be lost or stolen by intermediary connectors. ILP Prepare packets are sent with a condition or SHA-256 hash in them. When the receiver receives the packet, they present the fulfillment, or valid preimage of the hash, to execute the transfer.
+- **Connector** - Intermediary that maintains settlement accounts on one or more ledgers or payment systems. Connectors forward ILP packets. They may generate revenue from spreads on currency conversion, through subscription fees, or other means.
+- **ILP Packets** - ILPv4 has three packet types: Prepare, Fulfill, and Reject, which roughly correspond to request, response, and error messages. Connectors forward Prepare packets from senders to receivers and the connectors relay the Fulfill or Reject packets back from the receivers to the senders.
+- **Ledger** - A system that tracks and transfers value between accounts. In earlier versions of Interledger, ILP packets were transmitted through transfers on ledgers. In ILPv4, ledgers are used primarily to settle obligations between connectors, their peers, and their customers. 
+- **Payment Channel** - A method of using signed claims against funds held on a ledger, rather than on-ledger transfers that may be too slow or expensive, to settle obligations. The channels used in ILPv4 may be unidirectional or bidirectional but generally they will be [simple](../0022-hashed-timelock-agreements/0022-hashed-timelock-agreements.md#simple-payment-channels) rather than [conditional](../0022-hashed-timelock-agreements/0022-hashed-timelock-agreements.md#conditional-payment-channels-with-htlcs) channels.
+- **Transport and Application Layer Protocols** - Senders and receivers generally use higher-level protocols to agree on the condition and other details for each ILP packet. These protocols may also handle encryption and authentication of the data sent with ILP packets. For more details see the [Interledger Architecture overview](../0001-interledger-architecture/0001-interledger-architecture.md#interledger-protocol-suite) or the [Pre-Shared Key V2 (PSKv2)](../0025-pre-shared-key-2/0025-pre-shared-key-2.md) protocol.
+
+### Relation to Other Protocols
+
+ILPv4 packets may be sent over any communication channel between peers. Payment obligations created by ILP packets may be settled using any available means, ranging from ledger transfers triggered by an API, to signed updates to payment channels, or the physical delivery of cash or other goods.
+
+ILPv4 will probably be used with a Transport Layer protocol such as the [Pre-Shared Key V2 (PSKv2)](../0025-pre-shared-key-2/0025-pre-shared-key-2.md) protocol, which handles the generation of ILP conditions and fulfillments.
```

Comment:
This paragraph is ambiguous. Maybe change `ILPv4 will probably be used with...` to `In most use cases, ILPv4 will be used with...`